package.json

```diff
@@ -34,7 +34,8 @@
   "peerDependencies": {
     "react": "^15.0.0 || ^0.14.0",
     "react-dom": "^15.0.0 || ^0.14.0",
-    "react-motion": "^0.4.0"
+    "react-motion": "^0.4.0",
+    "warning": "^3.0.0"
```

Comment:
What about moving it in the dependency section?
This modules shouldn't have any impact in production.